src/shared/modules/udc/udcDuck.js

```diff
@@ -51,6 +67,7 @@ export const UPDATE_SETTINGS = `${NAME}/UPDATE_SETTINGS`
 export const UPDATE_DATA = `${NAME}/UPDATE_DATA`
 export const BOOTED = `${NAME}/BOOTED`
 export const METRICS_EVENT = `${NAME}/METRICS_EVENT`
+export const UPC_STARTUP = `${NAME}/STARTUP`
```

Comment:
APP_START seems to be sent before the app.jsx starts listening for metric events. From what I can see in mixpanel and my testing we don't get any APP_START events.. Now app.jsx sends a UDC startup event when it's ready to listen